src/main/java/edu/ucsb/cs156/happiercows/controllers/JobsController.java

```diff
@@ -64,4 +64,16 @@ public Job launchTestJob(
           });
     }
 
+    @ApiOperation(value = "Launch Job to Milk the Cows")
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
+    @GetMapping("/launch/testjob")
```

Comment:
Could you change this to a POST instead of a GET?

My earlier instructions may have been mistaken on this point, but this should be a POST request since it will change state on the server.

Reply:
done!